pkg/querier/queryrange/split_by_interval_test.go

```diff
@@ -440,6 +440,377 @@ func Test_splitByInterval_Do(t *testing.T) {
 	}
 }
 
+func Test_splitByInterval_Do_aligned(t *testing.T) {
+	ctx := user.InjectOrgID(context.Background(), "1")
+	data := []logproto.Entry{
+		{Timestamp: time.Unix(0, 0), Line: fmt.Sprintf("%d", 0)},
```

Comment:
Why don't we use a for loop to generate those ? what's the value in having them going back and forth in time ?